doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,171 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Data Container                                          | The data stored in a data container can be accessed through the liner offset on through objects described in metadata container.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Data unit                                               | A striping unit to which cluster-wide object or component data are mapped is called a data unit.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Units are only for global objects. COBs have blocks, which can have different size that units.